lib/checks/mobile/css-orientation-lock.js

```diff
@@ -130,8 +123,14 @@ function getIsOrientationLocked({ selectorText, style }) {
 		return false;
 	}
 
-	const locked = degrees % 90 === 0 && degrees % 180 !== 0;
-	return !!locked;
+	/**
+	 * When degree is a multiple of 180, it is not considered an orientation lock
+	 */
+	if (Math.abs(degrees) % 180 <= degreeThreshold) {
```

Comment:
The threshold only works if the number is slightly higher than 180, not if it ends up slightly below it.

```js
181 % 180 <= 1 // true
179 % 180 <= 1 // false
```